templates/vhost/_directories.erb

```diff
@@ -19,8 +19,11 @@
       <%- path = directory['path'] -%>
 
   <<%= provider %> "<%= path %>">
-      <%- if directory['headers'] -%>
-        <%- Array(directory['headers']).each do |header| -%>
+    <%- if directory['sendfile'] -%>
+    EnableSendfile <%= directory['sendfile'] %>
```

Comment:
```suggestion
    EnableSendfile <%= directory['enable_sendfile'] %>
```

Reply:
I selected `sendfile` because that name is used in the apache class, but I agree making it consistent with the actual option would be better. Fixed in the latest version.